client/src/services/sockets.js

```diff
@@ -45,7 +45,7 @@ class SocketManager {
     this.socket.on("message", (data) => {
       console.log("message received");
       this.subscribers.forEach((subscriber) => {
-        if (subscriber.name === "messages") {
+        if (subscriber.name == "messages") {
```

Comment:
is subscriber.name the name of user or it is "messages"?

Reply:
'messages'

Reply:
ok just the naming is a little confusing